### ISSUE TITLE
[DevTools Bug]: Can’t import chrome trace into react devtools

### ISSUE THREAD

**Issue by dinkinflickaa** [facebook/react#28767]
### Website or app

https://tsk9sq.csb.app/

### Repro steps

1. Go to the page and open devtools
2. Go to performance tab and start recording
3. Interact with the application
4. Stop recording
5. Download trace
6. Upload to react devtools
7. Expected - React timeline tab should display trace
8. Actual - React devtools fails to import the trace and shows an error - "e.sort is not a function. Try importing another Chrome performance profile."
![image](https://github.com/facebook/react/assets/5317985/a292d82b-36d0-4924-9e9d-1681b60fe17a)

I think its is an issue with how we are parsing the chrome trace [here](https://github.com/facebook/react/blob/e3ebcd54b98a4f8f5a9f7e63982fa75578b648ed/packages/react-devtools-timeline/src/import-worker/importFile.js#L22C1-L23C1).
Chrome outputs the trace file in the following format:
```json
{
  "traceEvents": [
    {"name": "Asub", "cat": "PERF", "ph": "B", "pid": 22630, "tid": 22630, "ts": 829},
    {"name": "Asub", "cat": "PERF", "ph": "E", "pid": 22630, "tid": 22630, "ts": 833}
  ],
  "displayTimeUnit": "ns",
  "systemTraceEvents": "SystemTraceData",
  "otherData": {
    "version": "My Application v1.0"
  },
  "stackFrames": {...}
  "samples": [...],
}
```
 
 but react devtools expects the file to contain only `traceEvents` array. 
React DevTools should be able to handle both formats because they are valid according to this document: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
 

### How often does this bug happen?

Every time

**Comment by abhayrsr** [comment]
Hi, I'm new to open source. would love to work on this bug.

**Comment by ant1m4tt3r** [comment]
indeed, had the same issue on my side. i believe it is a simple fix from what i saw in the codebase. are you working on it @abhayrsr ? let me know if you need help with that!
